CMSgov/design-system
# [WNMGDS-676] Update help drawer toggle to be a button

### Summary
Previously the help drawer toggle was a link, even though it is used like a button to activate the help drawer. This PR updates the toggle to be a transparent button, and adds a `ds-c-help-drawer__toggle` class

### How to test
- Ensure no visual regressions
- Test the toggle works well on screenreaders and IE11
